src/lib/modules/monitor/monitor.go

```diff
@@ -136,10 +125,19 @@ func (ps *Monitor) responseDelay(wg *sync.WaitGroup, route parser.Route) {
 	// Init paths for request-monitor-monitoring
 	path := PathReqResDelay + "/" + "chunk_req_res_" + routeSuffix + ".json"
 
-	c := make(chan utils.Response)
-	responseObject := routeDispatcher(route, c)
+	response := make(chan string)
+	req := request.New(route.URL, request.ToMap(route.Header), request.ToMap(route.Params), request.ToMap(route.Body))
+	// responseObject := routeDispatcher(route, c)
+	stamp := time.Now()
+	go req.Send(request.MethodUintPresentation(route.Method), response)
+	resp := <-response
+	c := utils.Response{
```

Comment:
rename this variable to something more appropriate because c is reserved for "channels"

Reply:
apparently, `c` is not required as well.i.e., without the use of a var, we can go ahead as well.